geodepy/tests/test_statistics.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+from geodepy import statistics
+from geodepy.statistics import np
+
+
+class TestStatistics(unittest.TestCase):
+    def test_rotation_matrix(self):
+        lat = 19.4792
+        lon = 70.6931
+
+        expected_result = np.array([
+            [-0.94376114, -0.11025276, 0.31170376],
+            [0.33062805, -0.31471096, 0.88974272],
+            [0.0,  0.94276261,  0.33346463],
+        ])
+
+        result = statistics.rotation_matrix(lat, lon)
+
+        np.array_equal(expected_result, result)
```

Comment:
Is this line needed? The result of this comparison is not used and the next line also does a comparison